src/pages/components/data-table/code.mdx

```diff
@@ -4,62 +4,222 @@ description: The data table component allows for the flexible display and sortin
 tabs: ['Code', 'Usage', 'Style', 'Accessibility']
 ---
 
-## Data table
+import { rowData, headerData } from '../../../data/components/data-table.js';
 
-<ComponentCode
-  name="Data table"
-  component="data-table"
-  variation="data-table"
-  hasReactVersion="datatable--default"
-  hasVueVersion="datatable--default"
-  hasAngularVersion="?path=/story/table--default"
-  codepen="jRWbZZ"></ComponentCode>
+## Default
 
-## Expandable data table
+<ComponentDemo
+  scope={{ rowData, headerData, React }}
+  links={{
+    React:
+      'http://react.carbondesignsystem.com/?path=/story/contentswitcher--default',
+    Angular:
+      'https://angular.carbondesignsystem.com/?path=/story/components-content-switcher--basic',
+    Vue:
+      'http://vue.carbondesignsystem.com/?path=/story/components-cvcontentswitcher--default',
+    Vanilla: 'https://the-carbon-components.netlify.com/?nav=content-switcher',
+  }}>{`<DataTable 
```

Comment:
I was also just messing around with this on a branch in my dev env before I saw this PR. I added these knobs to the default data table component demo:
```
  knobs={{
    DataTable: ['isSortable'],
    Table: ['size', 'useZebraStyles'],
  }}
```
The `isSortable` knob is working perfectly. However, I realized the Table knobs are being injected at the wrong place. Likely a bug in the regex. It's inserting the props in the middle of  `TableContainer` since it's the first "Table" string, so then it ends up looking like: 

```
Table useZebraStylesContainer
```
which doesn't work obviously. I can work on updating the regex and adding these knobs after this PR is merged if you want, or you can add it in this PR. Just thought it would be a good idea to have those props.

Reply:
Ah, yeah I gotta update the storybook links too.